### PR TITLE
Release 5.1.2

### PR DESCRIPTION
**Pull request by nicoddemus**

### CODE REVIEW

File: doc/en/assert.rst [hunk 1]
```diff
@@ -47,7 +47,7 @@ you will see the return value of the function call:
     E        +  where 3 = f()
 
     test_assert1.py:6: AssertionError
-    ============================ 1 failed in 0.02s =============================
+    ============================ 1 failed in 0.12s =============================
```
Comment (nicoddemus) [inline doc/en/assert.rst]: No more bullying from @asottile! 😆

Comment (asottile) [reply doc/en/assert.rst]: :snail: :snail: :snail:

File: doc/en/fixture.rst [hunk 2]
```diff
@@ -692,7 +692,7 @@ So let's just do another run:
     test_module.py:13: AssertionError
     ------------------------- Captured stdout teardown -------------------------
     finalizing <smtplib.SMTP object at 0xdeadbeef>
-    4 failed in 0.89s
+    4 failed in 1.69s
```
Comment (asottile) [inline doc/en/fixture.rst]: this one still snuck through though :laughing:

Comment (nicoddemus) [reply doc/en/fixture.rst]: 😢

Comment (nicoddemus) [reply doc/en/fixture.rst]: Unless there are further changes needed, I rather fix this in the next release. 😁

Comment (asottile) [reply doc/en/fixture.rst]: sg2m

**Pull request merged**